change/@microsoft-teams-js-2105a377-282c-452d-bb03-208852fcc2dd.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added optional `customeMessageIdStarter` parameter to `app.initialize` call to give an option to developers to set the starting number for messageIds. \\nHaving a high seed value for this field results in generating messageIds that have a less change of collision when there are multiple instances of teamsJS running by the application",
```

Comment:
```suggestion
  "comment": "Added optional `customeMessageIdStarter` parameter to `app.initialize` call to give an option to developers to set the starting number for messageIds. \\nHaving a high seed value for this field results in generating messageIds that have a less chance of collision when there are multiple instances of teamsJS running by the application",
```

Reply:
Also, if we do need to merge this, not sure if we want to be more specific about this being only used for internal Microsoft usage, or something like that.